src/transport/RendezvousSession.cpp

```diff
@@ -59,22 +55,6 @@ CHIP_ERROR RendezvousSession::Init(const RendezvousParameters & params, Transpor
     mSecureSessionMgr = sessionMgr;
     mAdmin            = admin;
 
-    // TODO: BLE Should be a transport, in that case, RendezvousSession and BLE should decouple
-    if (params.GetPeerAddress().GetTransportType() == Transport::Type::kBle)
-#if CONFIG_NETWORK_LAYER_BLE
-    {
-        ReturnErrorOnFailure(mParams.GetAdvertisementDelegate()->StartAdvertisement());
```

Comment:
This code has been removed. Do we not need the `AdvertisementDelegate` anymore ?

Reply:
I found we are still expected to enable BLE advertisement in RendezvousSession. Since BLE is only used for intial setup, I think it is OK so I added this line back.